grunttasks/bump.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// takes care of bumping the version number in package.json
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('bump', {
+    options: {
+      files: ['package.json', 'npm-shrinkwrap.json'],
+      bumpVersion: true,
+      commit: true,
+      commitMessage: 'Release v%VERSION%',
+      commitFiles: ['package.json', 'npm-shrinkwrap.json', 'CHANGELOG'],
+      createTag: true,
+      tagName: 'v%VERSION%',
+      tagMessage: 'Version %VERSION%',
+      push: false,
+      pushTo: 'origin',
+      gitDescribeOptions: '--tags --always --abrev=1 --dirty=-d'
+    }
+  });
```

Comment:
I think this bump config is pretty consistent in almost all our repos. Not sure if it's worth turning into its own module so you could just do something like `grunt.config('bump', require('mozilla-bump-config'));` so we can easily share this between repos. I guess the only risk would be if we remove npm-shrinkwrap.json from some repos, or have some inconsistencies between 'CHANGELOG' vs 'CHANGELOG.md'.

Bah, probably not worth it since I doubt we'll ever mass change these settings in the future.